gremlin-go/README.md

```diff
@@ -136,6 +136,16 @@ Note: The exact import name as well as the module prefix for `NewDriverRemoteCon
 		})
 ```
 
+## Aliases
+To make the code more readable and close to the Gremlin query language), you can use aliases
```

Comment:
Would it be useful if we provided a common list of alias to use? So people can just copy them? The only issue would be whether or not to capitalize the alias...

Reply:
added in https://github.com/lyndonbauto/tinkerpop/pull/107/commits/37df33c58afc5f26654b0be4713b0c8efec675a2